src/schematools/importer/base.py

```diff
@@ -287,6 +293,55 @@ def generate_db_objects(
                 db_schema_name,
             )
 
+    def load_view_sql(self, url: str) -> str:
+        """Load a view SQL file from a URL."""
+        if not url.startswith("http"):
+            raise ValueError(f"View URL {url} is not a valid URL")
+
+        response = requests.get(url)
```

Comment:
Normally I would expect here a `raise_for_status`. Or, it that not needed anymore to trigger exception raising?

Reply:
There are two mechanisms to create a view. One using SQLAlchemy and one using the Django ORM. It seems they do more or less the same? Or, are they complementary?

Reply:
Changing the function to use raise_for_status. 

I use the Django ORM in the Django management commands and the SQL Alchemy one in the CLI. It looks to me this is how others have done it.